metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -193,9 +194,11 @@ public Task<Void> ingest(@ActionParam(PARAM_ENTITY) @Nonnull Entity entity,
 
     SystemMetadata systemMetadata = populateDefaultFieldsIfEmpty(providedSystemMetadata);
 
-    // TODO Correctly audit ingestions.
-    final AuditStamp auditStamp =
-        new AuditStamp().setTime(_clock.millis()).setActor(Urn.createFromString(Constants.UNKNOWN_ACTOR));
+    Authentication authentication = AuthenticationContext.getAuthentication();
+    String actorUrnStr = authentication.getActor().toUrnStr();
+    // Getting actor from AuthenticationContext
+    log.debug(String.format("Retrieving AuthenticationContext for Actor with : %s", actorUrnStr));
```

Comment:
Let's remove this line!

Reply:
@jjoyce0510 added actor urn for openapi-servlet as well.

Reply:
awesome- thank you